pkg/controller/nodeipam/legacyprovider.go

```diff
@@ -59,7 +59,7 @@ func startLegacyIPAM(
 	if len(clusterCIDRs) > 1 {
 		klog.Warningf("Multiple cidrs were configured with FromCluster or FromCloud. cidrs except first one were discarded")
 	}
-	ipamc, err := ipam.NewController(cfg, kubeClient, cloud, cidr, serviceCIDR, nodeCIDRMaskSize)
```

Comment:
Should we log warning if len(nodeCIDRMaskSize) > 1?

Reply:
`nodeCIDRMaskSizes` is not user provided. When we generate the slice, we use the cluster cidr slice to determine and generate nodeCIDRMaskSizes. So I think just the warning for ignoring cluster cidrs should be good. WDYT?